TennisKata/TennisKata/Scoring/Game.swift

```diff
@@ -1,18 +1,52 @@
-//
-//  Copyright © 2018 NaMi. All rights reserved.
-//
-
 import Foundation
 
-public struct Player {
-    public let score: Int = 0
+public final class Player {
+    public var score: String = "0"
+    public var points: String!
+    public var playerOneScore: String = "0"
 
     public init() { }
+
+    public func scores(point: String) -> String {
+
+        switch point {
+        case "first":
+            points = "15"
+        case "second":
+            points = "30"
+        default:
+            points = "0"
+        }
+//        points = points + point
+        return points
+    }
+
+    public func getScores(for player: String) -> String {
+        return ""
+    }
+
 }
 
 public struct Game {
-    public let playerOne = Player()
+    public var playerOne = Player()
     public let playerTwo = Player()
 
     public init() { }
+
+    public func valid(score: String) -> Bool {
```

Comment:
I'm a bit confused by this. How can a game have a score? There are players, and they have scores, but a game does not have a score? And in your code, it only seems to have a single score, anyway? A game always has two scores - 15:0, 30:0 (also said as "30,Love", for example so I see where you are going with the strings, although this now means you have to do conversions between score strings and game strings, which seems confused - why not use numbers, which are easy to work with.

For example, modelling this will be much more difficult with string scores:

zero to three points are described as “love”, “fifteen”, “thirty”, and “forty” respectively.

Reply:
My idea was to first set the valid scoring rules of the game, hence method was added. But now I realise this is not required and only players will have scores. Agree and will modify code as per the review comments